config/src/lib.rs

```diff
@@ -203,6 +203,8 @@ pub struct Config {
     pub etherscan: EtherscanConfigs,
     /// list of solidity error codes to always silence in the compiler output
     pub ignored_error_codes: Vec<SolidityErrorCode>,
+    /// The minimum severity level that is treated as a compiler error
```

Comment:
Seems this comment is out of date
```suggestion
    /// When true, compiler warnings are treated as errors
```